client/src/components/AdvisorReports/AdvisorReportsTable.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Table } from 'react-bootstrap'
+import AdvisorReportsTableHead from 'components/AdvisorReports/AdvisorReportsTableHead'
+import AdvisorReportsRow from 'components/AdvisorReports/AdvisorReportsRow'
+import _uniqueId from 'lodash.uniqueid'
+
+const AdvisorReportsTable = (props) => {
+    let rows = props.data.map( (advisor) => {
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.